ethexe/contracts/test/Middleware.t.sol

```diff
@@ -78,6 +81,79 @@ contract MiddlewareTest is Test {
         assertEq(sym.networkMiddlewareService().middleware(address(middleware)), address(middleware));
     }
 
+    function test_election() public {
+        address[] memory operators = new address[](5);
+        address[] memory vaults = new address[](operators.length);
+
+        for (uint256 i = 0; i < operators.length; i++) {
+            operators[i] = address(uint160(i) + 0x1000);
+            _createOperator(operators[i]);
+            vaults[i] = _createVaultForOperator(operators[i]);
+        }
+
+        _depositFromInVault(owner, vaults[0], 1_000);
+        _depositFromInVault(owner, vaults[1], 2_000);
+        _depositFromInVault(owner, vaults[2], 1_000);
+        _depositFromInVault(owner, vaults[3], 5_000);
+        _depositFromInVault(owner, vaults[4], 1_000);
+
+        vm.warp(vm.getBlockTimestamp() + 1000);
+
+        vm.expectRevert(abi.encodeWithSelector(Middleware.IncorrectTimestamp.selector));
+        middleware.makeElectionAt(uint48(vm.getBlockTimestamp()), 10);
+
+        vm.expectRevert();
+        middleware.makeElectionAt(uint48(vm.getBlockTimestamp()) - 1, 0);
+
+        {
+            address[] memory res = middleware.makeElectionAt(uint48(vm.getBlockTimestamp() - 1), 1);
+            assertEq(res.length, 1);
+            assertEq(res[0], operators[3]);
+        }
+
+        {
+            address[] memory res = middleware.makeElectionAt(uint48(vm.getBlockTimestamp() - 1), 2);
+            assertEq(res.length, 2);
+            assertEq(res[0], operators[3]);
+            assertEq(res[1], operators[1]);
+        }
+
+        {
+            address[] memory res = middleware.makeElectionAt(uint48(vm.getBlockTimestamp() - 1), 3);
+            assertEq(res.length, 3);
+            assertEq(res[0], operators[3]);
+            assertEq(res[1], operators[1]);
+            assert(res[2] == operators[0] || res[2] == operators[2] || res[2] == operators[4]);
+        }
+
+        {
+            address[] memory res = middleware.makeElectionAt(uint48(vm.getBlockTimestamp() - 1), 4);
+            assertEq(res.length, 4);
+            assertEq(res[0], operators[3]);
+            assertEq(res[1], operators[1]);
+            assert(res[2] == operators[0] || res[2] == operators[2] || res[2] == operators[4]);
```

Comment:
assert - solidity built-in, assertTrue - foundry test-specific
```suggestion
            assertTrue(res[2] == operators[0] || res[2] == operators[2] || res[2] == operators[4]);
```

Reply:
Done